IDEA/build.gradle.kts

```diff
@@ -32,12 +32,18 @@ kotlin {
 
 dependencies {
     // From Kotlin documentation
-    implementation(kotlin("stdlib"))
-    implementation(kotlin("stdlib-jdk8"))
+    implementation("org.jetbrains.kotlin:kotlin-stdlib:1.4.10")
```

Comment:
For my own knowledge, is this the generally recommended way to depend on Kotlin, as opposed to the `kotlin` function? They mention [here](https://kotlinlang.org/docs/reference/using-gradle.html#set-dependencies-on-test-libraries) that `kotlin(...)` is a shorthand for what you've got here. I don't care either way, just curious 🙂

Reply:
Because this way you can give an explicit version. I have no idea if this is the recommended way, though, honestly, I would expect this version to be taken from `id("org.jetbrains.kotlin.jvm") version "1.4.10"` in the plugins block when you use `kotlin(...)` here, but I have no idea if that is the case. I did state the explicit version here to try to overcome the warning mentioned earlier.